### ISSUE TITLE
Polar encoder and decoder with factory classes

### ISSUE THREAD

**Issue by Aishusiva** [aff3ct/aff3ct#92]
Can you please help me on how to use polar encoder and decoder with factory classes.

**Comment by SubtleMuffin** [comment]
Did you search any previous closed issues / pull requests both here and aff3ct/my_project_with_aff3ct?

**Comment by Aishusiva** [comment]
Yes I did. In all the cases polar encoding and decoding has been used only in bootstrap code.
